README.md

```diff
@@ -5,6 +5,22 @@
 # libopenvizsla
 An attempt to reimplement [OpenVizsla](http://openvizsla.org/) host software in plain C.
 
+## Getting Started
+This section describes how to get started with OpenVizsla and Wireshark. \
+**IMPORTANT NOTICE:** As of late 2023, master is broken (see [#30](https://github.com/matwey/libopenvizsla/pull/30#issuecomment-1501915611)). Use Tag/Release 0.1.1 instead.
+
+### Linux
+1. Build the project as described below (use Tag 0.1.1 as of late 2023)
+2. Copy the `ovextcap` executable to Wiresharks `extcap` directory
+3. Run Wireshark and start capturing
+
+### Windows
+1. Make sure `FTDI D2XX` drivers are installed
```

Comment:
There is no need for FTDI drivers because libusbK is being used. The FTDI drivers may only be necessary if you are building brand new OpenVizsla and have to program the EEPROM (but this is not supported here, only in ov_ftdi python code).

Reply:
Thanks for the hint, removed!